app/models/megaphone/episode.rb

```diff
@@ -16,15 +19,19 @@ class Episode < Integrations::Base::Episode
 
     CREATE_ATTRIBUTES = CREATE_REQUIRED + %i[pubdate pubdate_timezone author link explicit draft
       subtitle summary background_image_file_url background_audio_file_url pre_count post_count
-      insertion_points guid expected_adhash original_filename original_url
-      episode_number season_number retain_ad_locations advertising_tags ad_free]
+      insertion_points guid expected_adhash original_filename episode_number season_number
+      retain_ad_locations ad_free episode_type clean_title]
```

Comment:
`original_url` is for imports, don't update it
`episode_type` was missing
`clean_title` was missing

`advertising_tags` will be a different PR, don't update for now